development/client/tsconfig.json

```diff
@@ -16,7 +16,8 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react"
+    "jsx": "react",
+    "allowSyntheticDefaultImports": true
```

Comment:
Ok, I think we've removed this twice now... I think `create-react-app` might be auto-generating this?